Reenable Rowan Collateral

AustinoBombino opened the pull request:
reenable rowan collateral for testing on staging

New TX form: `sifnoded tx margin update-rowan-collateral --enabled=true/false `

AustinoBombino commented:
@timlind @canercandan @Jedi2002 
added a toggle parameter and tx

canercandan commented:
@AustinoBombino nice thanks for working on this, I believe we should add a migration handler as well to set the default value for this new toggle correct?

AustinoBombino commented:
> @AustinoBombino nice thanks for working on this, I believe we should add a migration handler as well to set the default value for this new toggle correct?

@canercandan Yup, fixed the tests and added a migrator

canercandan reviewed (approved):
LGTM

Jedi2002 commented on x/margin/keeper/msg_server.go:
```diff
@@ -344,6 +344,23 @@ func (k msgServer) UpdatePools(goCtx context.Context, msg *types.MsgUpdatePools)
 	return &types.MsgUpdatePoolsResponse{}, nil
 }
 
+func (k msgServer) UpdateRowanCollateral(goCtx context.Context, msg *types.MsgUpdateRowanCollateral) (*types.MsgUpdateRowanCollateralResponse, error) {
```
We can probably change the update params logic , instead of adding new transactions for each param ?
